jib-core/build.gradle

```diff
@@ -79,6 +79,13 @@ tasks.withType(JavaCompile) {
     }
 }
 
+// Fail build on javadoc warnings
+if (JavaVersion.current().isJava8Compatible()) {
+    tasks.withType(Javadoc) {
+        options.addStringOption('Xwerror', '-quiet')
```

Comment:
What does the `-quiet` do?

Reply:
I think it's just a hack to fill the second argument of `addStringOption`, since gradle automatically adds the '-quiet' option anyway.

Reply:
It looks like you could possibly try `.addBooleanOption('Xwerror', true)`

https://github.com/gradle/gradle/issues/1393#issuecomment-280287314